activerecord/test/cases/helper.rb

```diff
@@ -57,6 +57,20 @@ def supports_default_expression?
   end
 end
 
+def reset_callbacks(klass, kind)
```

Comment:
Add new helper method in the `test/cases/test_case.rb`.
https://github.com/rails/rails/blob/master/activerecord/test/cases/test_case.rb